daemon/src/maker_cfd.rs

```diff
@@ -579,7 +579,7 @@ where
                 taker_id,
                 command: TakerCommand::NotifyOrderAccepted { id: order_id },
             })
-            .await?;
+            .await??;
```

Comment:
Is it not as a separate commit already? Can't really split it further, without introducing and removing the spawn fallible in a commit it-between. 

IMHO we really should stop the contract setup if it fails instead of just logging it.

Reply:
I see two commits:

1. Removing log_error and use spawn_fallible on the calling side

2. Use xtra_productivity to remove boilerplate.

I would like to retain functionality for now, feel free to propose a follow-up where we discuss changing the behaviour.

Reply:
ok, I'll create a follow-up with that simplification separately.

Reply:
Started #418